docs/02_cleos/02_how-to-guides/how-to-deploy-a-smart-contract.md

```diff
@@ -13,7 +13,10 @@ Deploy an EOSIO contract
 Execute:
 
 ```sh
-cleos set contract contract_account contract_directory_path
+cleos set contract contract_account contract_folder [wasm-file] [abi-file]
 ```
 
-Replace the `contract_directory_path` with the path points to your contract folder
+Replace the `contract_folder` with the path that points to your contract folder
+
+[[info | Default contract name]]
+| By default, `cleos` treats the last folder specified in `contract_folder` as the contract name. Therefore, it expects `.wasm` and `.abi` files with that contract name as filename. This can be overridden with the optional `wasm-file` and `abi-file` parameters.
```

Comment:
This should not say "last", I think that makes it confusing. An example would be good.

Reply:
I will add an example to clarify.